spec/classes/init_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 describe 'nslcd' do
 
-  {'Ubuntu' => 'Debian', 'Debian' => 'Debian'}.each do |system, family|
+  {'RedHat' => 'Debian', 'Ubuntu' => 'Debian', 'Debian' => 'Debian'}.each do |system, family|
```

Comment:
@werekraken added support for RedHat-based OS in PR #1. So that can help simplify your PR too :)